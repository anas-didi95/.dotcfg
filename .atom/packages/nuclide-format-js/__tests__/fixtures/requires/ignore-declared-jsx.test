const Components = require('Components');

var {A, B} = Components;

var jsx = <A><B/></A>;
