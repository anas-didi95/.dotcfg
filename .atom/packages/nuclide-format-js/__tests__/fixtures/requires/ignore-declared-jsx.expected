const Components = require('Components');
const React = require('React');

var {A, B} = Components;

var jsx = <A><B/></A>;
